src/lib/styles/components/_index.scss

```diff
@@ -4,3 +4,4 @@
 @forward "../components/button";
 @forward "../components/inputs";
 @forward "../components/card";
+@forward "../components/header";
```

Comment:
Here you can see clearly how header is different with the components we are working in this folders.
All the other components are super repetitive, we have many cards, inputs and buttons all over the UI.
As they aren't too complex to make a specific component for them, we just drop some styling for them here.
(Probably we'll want to take out button in the future and make specific component because it can get more complex).

But header is not repetitive and is much more complex (needs JS). So it's much better to make a component and put everything related to the Header there...